content/reference/user/private_locations/index.md

```diff
@@ -0,0 +1,104 @@
+
+{{< alert info >}}
+Only two outbounds domain must be allowed in your network:
+- The API Gateway: `api.gatling.io` on port `443`
+- Download of the Gatling simulation on S3: `eu-west-3.amazonaws.com` on port `443`
+{{< /alert >}}
+
+## Control plane configuration
+
+### Control plane token
+
+Access the private locations section by clicking on the Private locations in the navigation bar (only visible if the feature is activated on your organization).
+
+Click on create **Control Plane**, and fill an unique identifier. (only lowercase, separated by underscores)
+
+{{< img src="create-control-plane.png" alt="Create control plane" >}}
+
+On the next modal, make sure to copy the **Control Plane** token, you’ll need it later.
+
+{{< img src="copy-token.png" alt="Copy control plane token" >}}
+
+You can see the **Control Plane** you just created under the Control planes tab.
+
+{{< img src="control-planes-uninitialized.png" alt="Control Planes table" >}}
+
+It's defined by an unique identifier, a description (provided later on by its configuration) and a status.
+
+Status have three possible value:
+- **Uninitialized:** Control plane never reached the application.
+- **Up:** Control plane is properly configured, uploaded its private locations configuration, and reaches periodically the application to fetch new runs.
+- **Down:** Control plane has been up, but didn't reach the application for a while.
+
+### Control plane deployment
+
+Control plane agent is distributed as docker image: `gatlingcorp/control-plane`
+And are configured by a file mounted at `/app/conf/control-plane.conf`.
+
+{{< alert info >}}
+Configuration use [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) (Human-Optimized Config Object Notation):
+{{< /alert >}}
+
+Example with an AWS pool (only supported type at the moment):
+{{< include-code "control-plane.conf" hocon >}}
+
+Only **Control Plane** and locations ID, description and type are transferred to Gatling Enterprise Cloud.
+
+## Managing Control Planes on Gatling Enterprise Cloud
+
+Once configured, your **Control Plane** status should be `Up`, details are available by clicking on the {{< icon eye >}} button.
+Token can be refresh by clicking on the {{< icon undo >}} button.
+
+{{< img src="control-plane-details.png" alt="Control Planes details" >}}
+
+Private locations can be seen in the Locations tab.
+You can see their relations to their **Control Plane** and simulations.
+
+{{< img src="locations-table.png" alt="Locations table" >}}
+
+They're deletable if they're neither linked to a **Control Plane** that is not `Up` and any simulation.
+
+# Simulation configuration
+
+When configuring a simulation, on the locations step, click on private.
+
+{{< img src="simulation-config.png" alt="Simulation configuration of private locations" >}}
+
+{{< alert info >}}
+At the moment, it's not possible to use private locations along with public ones.
+{{< /alert >}}
+
+# Credit consumption
+
```

Comment:
Next file (content/reference/user/simulations/images/create-simulation-locations.png) should not be updated right now.

We don't know yet how to show the new feature for those who don't have access to it